### PR TITLE
implement theme support

### PR DESCRIPTION
**Pull request by daniele-zurico**

### CODE REVIEW

File: src/app/application.ts [hunk 1]
```diff
@@ -328,22 +328,30 @@ export namespace Application {
     let processResources = () => {
         logger.info('Copy main resources');
         fs.copy(path.resolve(__dirname + '/../src/resources/'), path.resolve(process.cwd() + path.sep + defaultFolder), function (err) {
-            if (err) {
-                logger.error('Error during resources copy');
-            } else {
-                if (program.extStyle) {
-                    fs.copy(path.resolve(process.cwd() + path.sep + program.extStyle), path.resolve(process.cwd() + path.sep + defaultFolder + '/styles/bootstrap.min.css'), function (err) {
+            if(err) {
+                logger.error('Error during external styling file copy ', err);
```
Comment (daniele-zurico) [inline src/app/application.ts]: is still there. It has been moved on line 332. the logic is:
1) I copy everything
2) I check if extTheme is present. If yes I remove the copied styles folder and I copy the newOne

Comment (daniele-zurico) [inline src/app/application.ts]: you right I changed the comment in a wrong way

**Comment by vogloblinsky** [comment]
@daniele-zurico I merged your PR tonight too quickly, and i don't understand well why you force user providing all css files (reset.css, etc...) ?
The css framework is bootstrap.
Using one of this themes is easy : [bootswatch](http://bootswatch.com/)
Just download the bootstrap.min.css and supply it to compodoc with -y flag
The first draft i implemented was to override the default bootstrap file. Just that.
Sorry but it is more clear in that way. I revert tomorrow if it is ok for you.

**Comment by daniele-zurico** [comment]
Yo! the point is that everyone should be free to override the style, create his specific version of bootstrap. You can revert if you want but the point is that in your hbs file you specify exactly wich css file you want to load and if I insert a new one called for example daniele.css how we can load it? The example that I did was to change the color of the top bar left bar and in the copyright div add a background image that is my logo. Are we able to achieve the same result with your change?

**Comment by vogloblinsky** [comment]
Ok i understand ;)
I will update the README in that way.

**Comment by daniele-zurico** [comment]
what about if we leave both solutions? One is to override just the style and I agree with you the other is to create custom themes

**Comment by vogloblinsky** [comment]
Your solution is more open. A clear explanation in README will be enougth.

**Comment by vogloblinsky** [comment]
Just refactor it to override the default style folder with supplied folder, and not delete the default one.
Release coming soon.

**Pull request merged**